lib/main_publiccloud.pm

```diff
@@ -202,8 +202,9 @@ sub load_publiccloud_tests {
         loadtest 'boot/boot_to_desktop';
         if (get_var('PUBLIC_CLOUD_MIGRATION')) {
             my $args = OpenQA::Test::RunArgs->new();
-            loadtest('publiccloud/upload_image', run_args => $args);
-            loadtest('publiccloud/migration', run_args => $args);
+	    loadtest('publiccloud/upload_image', run_args => $args);
+            loadtest('publiccloud/azure_vm_cli', run_args => $args);
```

Comment:
this can not be here , it should be executed at `elsif (check_var('PUBLIC_CLOUD_TOOLS_CLI', 1)) {` branch

Reply:
I need to discuss with you to discuss the current approach. I have created separate script azure_vm_cli.pm. Not integrating with the existing azure_cli.pm. Just temporary I have added under DMS migration for me to continue the testing.